core/src/components/text-area/text-area.stories.js

```diff
@@ -0,0 +1,143 @@
+    </div>
+  </div>
+</div>
+`;
+
+export const Disabled = () => `
+<div class="fudis-text-area fudis-input-size__lg">
+  <label class="fudis-label" for="fudis-text-area-id">
+    <span class="fudis-label__content">
+      <span class="fudis-label__content__text">Text area label</span>
+    </span>
+  </label>
+  <textarea id="fudis-text-area-id" class="fudis-form-input fudis-text-area__input" aria-describedby="fudis-text-area-id_guidance" aria-disabled="true" disabled></textarea>
+  <div id="fudis-guidance-id" class="fudis-guidance">
+    <div id="fudis-text-area-id_guidance">
+      <div id="fudis-guidance-id-errors" class="fudis-guidance__errors">
+        <div class="fudis-guidance__errors__list"></div>
+      </div>
+      <p class="fudis-guidance__help-text">Guidance text</p>
+    </div>
+  </div>
+</div>
+`;
+
+export const Invalid = () => `
+<div class="fudis-text-area fudis-input-size__lg">
+  <label class="fudis-label" for="fudis-text-area-id">
+    <span class="fudis-label__content">
+      <span class="fudis-label__content__text">Text area label</span>
+    </span>
+  </label>
+  <textarea id="fudis-text-area-id" class="fudis-form-input fudis-form-input--touched fudis-text-area__input" aria-describedby="fudis-text-area-id_guidance" aria-invalid="true"></textarea>
+  <div id="fudis-guidance-id" class="fudis-guidance">
+    <div id="fudis-text-area-id_guidance">
+      <div id="fudis-guidance-id-errors">
+        <div class="fudis-guidance__errors__list">
+          <p class="fudis-error-message fudis-error-message__form-error">Validator error message</p>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+`;
+
+export const CharacterLimit = () => `
+<div class="fudis-text-area fudis-input-size__lg">
+  <label class="fudis-label" for="fudis-text-area-id">
+    <span class="fudis-label__content">
+      <span class="fudis-label__content__text">Text area label</span>
+    </span>
+  </label>
+  <textarea id="fudis-text-area-id" class="fudis-form-input fudis-text-area__input" aria-describedby="fudis-text-area-id_guidance"></textarea>
+  <div id="fudis-guidance-id" class="fudis-guidance">
+    <div id="fudis-text-area-id_guidance">
+      <div id="fudis-guidance-id-errors" class="fudis-guidance__errors">
+        <div class="fudis-guidance__errors__list"></div>
+      </div>
+      <p class="fudis-guidance__help-text">Guidance text</p>
+    </div>
+    <small class="fudis-guidance__character-limit-indicator fudis-guidance__character-limit-indicator__lg">
+      0/100
+      <span class="fudis-visually-hidden">characters used</span>
+    </small>
+  </div>
+</div>
+`;
+
+export const Required = () => `
+<div class="fudis-text-area fudis-input-size__lg">
+  <label class="fudis-label" for="fudis-text-area-id">
+    <span class="fudis-label__content">
+      <span class="fudis-label__content__text">Text area label</span>
+      <span class="fudis-label__content__space"></span>
```

Comment:
This causes wider spacing as expected. Somehow related to white-space: break-spaces style in fudis-label__content.